Doc/library/stdtypes.rst

```diff
@@ -2333,7 +2333,11 @@ String objects have one unique built-in operation: the ``%`` operator (modulo).
 This is also known as the string *formatting* or *interpolation* operator.
 Given ``format % values`` (where *format* is a string), ``%`` conversion
 specifications in *format* are replaced with zero or more elements of *values*.
-The effect is similar to using the :c:func:`sprintf` in the C language.
+The effect is similar to using the :c:func:`sprintf` in the C language. For
+example::
```

Comment:
I prefer to add stuff like this as doctests, so we're sure our examples actually work.

```suggestion
The effect is similar to using the :c:func:`sprintf` function in the C language.
For example:

.. doctest::
```

Reply:
Also, I find the example a little bit contrived, but I haven't got a better suggestion ATM :(

Reply:
I didn't know this directive. I will use it more now.

About the example, well... almost all are contrived. 😅